README.md

```diff
@@ -24,3 +24,54 @@ end
 3. Commit your changes (`git commit -am 'Add some feature'`)
 4. Push to the branch (`git push origin my-new-feature`)
 5. Create new Pull Request
+
+## Testing
+
+### Installing the Rubies
+
+For setting up a dev environment with all the Rubies on it, it is convenient to
+use the [rake-compiler-dev-box](https://github.com/tjschuck/rake-compiler-dev-box).
```

Comment:
This is nothing we've ever used thus I wouldn't feel comfortable including it in our readme.